chart/templates/mayastor/agents/core/agent-core-deployment.yaml

```diff
@@ -50,6 +50,7 @@ spec:
             - "--pool-commitment={{ .Values.agents.core.capacity.thin.poolCommitment }}"
             - "--volume-commitment-initial={{ .Values.agents.core.capacity.thin.volumeCommitmentInitial }}"
             - "--volume-commitment={{ .Values.agents.core.capacity.thin.volumeCommitment }}"
+            - "--faulted-child-wait-period={{ .Values.agents.core.fast_rebuild_wait_period }}"
```

Comment:
Name the cmdline argument similar to the helm field, refer the above volume-commitment example.

Reply:
I thought of keeping it same but "faulted child wait" may not be as easy to comprehend for users compared to a more abstracted "fast rebuild wait" ?

Reply:
The field description and helm documentation would be sufficient imo

Reply:
Alright.